lib/full-site-editing/full-site-editing.php

```diff
@@ -20,7 +20,7 @@ function gutenberg_is_fse_theme() {
  * @return boolean Whether the current theme is FSE-enabled or not.
  */
 function gutenberg_supports_block_templates() {
-	return gutenberg_is_fse_theme() || ! current_theme_supports( 'disable-full-site-editing' );
+	return gutenberg_is_fse_theme() || current_theme_supports( 'full-site-editing' );
```

Comment:
Sorry to bother you again :P do you think the support flag should be more `block-templates`  instead of `full-site-editing` ?

Reply:
I was thinking about that too, and initially, I was a bit torn... The `gutenberg_supports_block_templates` is used in lots of places (replacing `gutenberg_is_fse_theme`), and it does turn on/off many aspects of FSE.
But I think you're right, using `block-templates` would make more sense. In a classic theme templates will be the only thing enabled, and in an FSE theme the support is irrelevant. Changed 👍